src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx

```diff
@@ -36,8 +37,33 @@ export default class PatientViewStructuralVariantTable extends React.Component<
     IPatientViewStructuralVariantTableProps,
     {}
 > {
+    @observable oncokbWidth = 22;
+    private oncokbInterval: Timeout;
+
     constructor(props: IPatientViewStructuralVariantTableProps) {
         super(props);
+        makeObservable(this);
+
+        this.oncokbInterval = setInterval(() => {
+            if (
+                document
+                    .getElementById('sv-annotation')
+                    ?.getElementsByClassName('oncokb-content')[0]?.clientWidth
+            ) {
+                this.oncokbWidth =
+                    Number(
+                        document
+                            .getElementById('sv-annotation')
+                            ?.getElementsByClassName('oncokb-content')[0]
+                            ?.clientWidth
+                    ) || 22;
+                clearInterval(this.oncokbInterval);
+            }
```

Comment:
should we extract this to a utility function? this is very similar to the one for `copy-number-annotation`

Reply:
@onursumer It could work but would it introduce some other code? Because we need to have `clearInterval(this.oncokbInterval);`, so for example if we move them to a util function, then we will have something like this:
```
this.oncokbInterval = setInterval(() => {
            someUtilFunction(this.setOncokbWidth, this.clearIntervalFn);
        }, 500);
```
and we will have to add this wherever that util function is called:
```
    @action.bound
    private clearIntervalFn() {
        clearInterval(this.oncokbInterval);
    }

    @action.bound
    private setOncokbWidth(width: number) {
        this.oncokbWidth = width;
    }
```

Reply:
Let's not worry about this then for now.